hack/go-version-check.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# List or check that the golang version embedded in files (specifically our Dockerfile)
+# is using a supported version of Go.
+# Run with --show to list the images the script thinks are current.
+# Run with --check (the default) to verify the Dockerfile contains a current
+# image & version tag.
+#
+
+set -e
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+PROJECT_DIR="$(cd "${SCRIPT_DIR}/.." && pwd)"
```

Comment:
Consider change to:
```
PROJECT_DIR="$(readlink -f "${SCRIPT_DIR}/..")"
```

It avoids the extra call to `pwd`.

Reply:
done